modules/compiler/spirv-ll/source/builder_core.cpp

```diff
@@ -3010,9 +3010,7 @@ cargo::optional<Error> Builder::create<OpPtrAccessChain>(
       basePointeeTy, base, indexes, module.getName(op->IdResult()),
       IRBuilder.GetInsertBlock());
 
-  if (basePointeeTy->isStructTy()) {
-    checkMemberDecorations(basePointeeTy, indexes, op->IdResult());
-  }
+  checkMemberDecorations(basePointeeTy, indexes, op->IdResult());
```

Comment:
Is it valid to call `checkMemberDecorations` when the type isn't a struct type? This seems like a change in behaviour.

Reply:
I've reverted now